install.html

```diff
@@ -210,101 +197,16 @@
-            Channel</a> with WebNN flag enabled in <code>about:flags</code>
-          <ol style="list-style: lower-alpha;">
-            <li>Download from <a
-                href="https://www.microsoft.com/en-us/edge/download/insider">https://www.microsoft.com/en-us/edge/download/insider</a>
-            </li>
-            <li>Run installer</li>
-            <li>Navigate to <code>about://flags</code></li>
-            <li>Search for <code>Enables WebNN API</code> and change it to <code>Enabled</code></li>
-            <li>Exit browser</li>
-          </ol>
-        </li>
-        <li>
-          Download DirectML redistributable
-          <ol style="list-style: lower-alpha;">
-            <li>Download DirectML from <a
-                href="https://www.nuget.org/packages/Microsoft.AI.DirectML/1.15.4">https://www.nuget.org/packages/Microsoft.AI.DirectML/1.15.4</a>
-            </li>
-            <li>Rename <code>microsoft.ai.directml.1.15.4.nupkg</code> to
-              <code>microsoft.ai.directml.1.15.4.nupkg.zip</code> and extract it</li>
-          </ol>
-        </li>
-        <li>
-          Copy <code>microsoft.ai.directml.1.15.4.nupkg.zip\bin\&lt;arch&gt;-win\directml.dll</code> to the appropriate
-          directory<br>(replace with x64 on Intel devices and arm64 on Qualcomm devices)
-          <ol style="list-style: lower-alpha;">
-            <li>Edge Dev: <code>C:\Program Files (x86)\Microsoft\Edge Dev\Application\129.0.2779.0\</code>
-              <ol>
-                <li>When the dialog asks for Administrator permission, choose <code>Continue</code></li>
-              </ol>
-            </li>
-            <li>Edge Canary: <code>%LOCALAPPDATA%\Microsoft\Edge SxS\Application\129.0.2779.0\</code></li>
-            <li>Note the following on copying <code>Directml.dll</code> to Edge directory
-              <ol>
-                <li>The version-specific directory (129.0.2779.0) may differ on your machine</li>
-                <li>New versions of Edge may require <code>directml.dll</code> to be recopied to the directory</li>
-              </ol>
-            </li>
-          </ol>
-        </li>
-        <li>
-          Launch Edge insider
-          <ol style="list-style: lower-alpha;">
-            <li>Open terminal and change your working directory to the Edge Insider build:
-              <ul>
-                <li>If using Edge Dev: <code>C:\Program Files (x86)\Microsoft\Edge Dev\Application</code></li>
-                <li>If using Edge Canary: <code>%LOCALAPPDATA%\Microsoft\Edge SxS\Application</code></li>
-              </ul>
-            </li>
-            <li><code>.\msedge.exe --use-redist-dml --disable_webnn_for_npu=0 --disable-gpu-sandbox</code></li>
-          </ol>
-        </li>
-      </ol>
+      For the latest and most reliable installation instructions, please refer to the <a
+        href="https://webnn.io/en/learn/get-started/installation">WebNN Installation
```

Comment:
Rather than including the installation directions here, the new text points to this other link, which is fine, but following that link, I see directions "It’s recommended to install the latest Intel® Arc", which only makes sense *if* your GPU is Intel-based (meaning it wouldn't work on the desktop I'm typing this comment on now). So that page should have at least some qualifying text contingent on the hardware you have, if WebNN atop GPU+WinMLORT currently only works on Intel GPU's, that should be stated there too.

Reply:
Yeah, I'm not expecting steps there for any other IHV's until those steps are known, just a clarification on that page the instructions only apply *if* the user has an Intel GPU.